projects/angular/src/layout/_card.clarity.scss

```diff
@@ -169,27 +188,72 @@
     }
   }
 
-  .card-header,
-  .card-block,
-  .card-footer {
-    padding: card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal;
+  .card-block {
+    @include mixins.css-var(
+      padding,
+      clr-card-block-padding,
+      card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
+      variables.$clr-use-custom-properties
+    );
   }
 
-  .card-header,
-  .card-title {
+  .card-footer {
     @include mixins.css-var(
-      color,
-      clr-card-title-color,
-      card-variables.$clr-card-title-color,
+      padding,
+      clr-card-footer-padding,
+      card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
       variables.$clr-use-custom-properties
     );
+  }
+
+  h3 {
+    &.card-header {
```

Comment:
This could be simplified to just `h3.card-header` since there isn't other nested rules. Same for the `h4` one.